doc/whats_new.rst

```diff
@@ -40,6 +40,7 @@ Detailed list of changes
 ^^^^^^^^^^^^^^^
 
 - :class:`~mne_bids.BIDSPath` now supports the new ``"sessions"`` suffix, by `Jonathan Vanhoecke`_ and `Richard Höchenberger`_ (:gh:`1137`)
+- The :func:`~mne_bids.BIDSPath.rm` method will safely delete all the files compatible with that path and update the ``scans.tsv`` and ``participants.tsv`` files accordingly, by `Pierre Guetschel`_ and `Adam Li`_ (:gh:`1149`)
```

Comment:
@PierreGtch you need to add yourself some lines above to:

> The following authors contributed for the first time. Thank you so much! 🤩

you furthermore need to add yourself to these two files:

- https://github.com/mne-tools/mne-bids/blob/main/doc/authors.rst
- https://github.com/mne-tools/mne-bids/blob/eeb59034694df365cbdb49a66c1716c7e8d75441/CITATION.cff#L162 (right after Jonathan Vanhoecke)

Reply:
Thanks, done